src/vm/jitinterface.cpp

```diff
@@ -12570,7 +12572,7 @@ CorJitResult CallCompileMethodWithSEHWrapper(EEJitManager *jitMgr,
         flags.Set(CORJIT_FLAGS::CORJIT_FLAG_FRAMED);
     if (g_pConfig->JitAlignLoops())
         flags.Set(CORJIT_FLAGS::CORJIT_FLAG_ALIGN_LOOPS);
-    if (ftn->IsVersionableWithJumpStamp() || g_pConfig->AddRejitNops())
+    if (!ftn->IsVersionableWithoutJumpStamp() || g_pConfig->AddRejitNops())
```

Comment:
Is this change an improvement? I think this change will add the jump stamps to more methods - even the ones that do not need it.

Reply:
It removes most all jump stamps from ngen/crossgen codegen.
It might not be safe to check this in.

Reply:
In System.Private.Corelib.ni.dll it contains 29,544 methods (when all methods are crossgen-ed)
Of those methods 46.8% have an extra **`nop`** in the method prolog to pad it's size to be 5-bytes.

Reply:
I will change this to:

` if (!ftn->IsVersionableWithoutJumpStamp() && ftn->IsVersionableWithJumpStamp()) || g_pConfig->AddRejitNops())`